Add 16.1 release notes (#9873)
chef/chef: RELEASE_NOTES.md

This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.

# Chef Infra Client 16.1

## Ohai 16.1

Ohai 16.1 includes a new `Selinux` plugin which exposes `node['selinux']['status']`, `node['selinux']['policy_booleans']`, `node['selinux']['process_contexts']`, and `node['selinux']['file_contexts']`. Thanks [@davide125](http://github.com/davide125) for this contribution. This new plugin is an optional plugin, which is disabled by default. It can be enabled within your `client.rb`:

```ruby
ohai.optional_plugins = [ :Selinux ]

> ]

Reviewer: ```suggestion
```

```

## Chef InSpec 4.18.114

> InSpec has been updated from 4.18.111 to 4.18.114, which adds new `--reporter_message_truncation` and `--reporter_backtrace_inclusion` reporter options to truncate message and suppress stacktraces.

Reviewer: ```suggestion
InSpec has been updated from 4.18.111 to 4.18.114, which adds new `--reporter_message_truncation` and `--reporter_backtrace_inclusion` reporter options to truncate messages and suppress backtraces.
```

If the option name uses backtrace the description should as well?

## Debian 10 aarch64

> Chef Infra Client packages are now produces for Debian 10 on the aarch64 architecture. These packages are available at [downloads.chef.io](https://downloads.chef.io/chef/)

Reviewer: ```suggestion
Chef Infra Client packages are now produced for Debian 10 on the aarch64 architecture. These packages are available at [downloads.chef.io](https://downloads.chef.io/chef/).
```

## Bug Fixes

- Resolved a regression in the `launchd` resource that would cause it to fail to converge.
- The `:disable` action in the `launchd` resource no longer fails if the plist was not found.
- Several Ruby 2.7 deprecation warnings have been resolved.

# Chef Infra Client 16.0.287

The Chef Infra Client 16.0.287 includes important bug fixes for the Chef Infra Client 16 release:

- Fixes the failure to install Windows packages on the 2nd convergence of the Chef Infra Client.
- Resolves several failures in the `launchd` resource.
- Removes an extra `.java` file on Windows installations that would cause a failure in the IIS 8.5 Server Security Technical Implementation Guide audit.

> - Updates the `windows_printer` so that the driver property will only be required when using the `:create` action.

Reviewer: ```suggestion
- Updates the `windows_printer` resource so that the driver property will only be required when using the `:create` action.
```
